frontend/packages/kubevirt-plugin/src/constants/vm/constants.ts

```diff
@@ -25,7 +25,7 @@ export const TEMPLATE_DATAVOLUME_NAME_PARAMETER = 'SRC_PVC_NAME';
 export const TEMPLATE_DATAVOLUME_NAMESPACE_PARAMETER = 'SRC_PVC_NAMESPACE';
 export const TEMPLATE_VM_DOMAIN_LABEL = 'kubevirt.io/domain';
 export const TEMPLATE_VM_SIZE_LABEL = 'kubevirt.io/size';
-export const TEMPLATE_VM_GOLDEN_OS_NAMESPACE = 'openshift-kubevirt-base-images';
+export const TEMPLATE_VM_GOLDEN_OS_NAMESPACE = 'openshift-virtualization-os-images';
```

Comment:
Aren't you supposed to fetch this namespace name from our templates instead of declaring it yourselves?

Reply:
In the UI we list all the OSs with their os PVC, so we need to loop over each OS, parse the name, namespace for the PVC and check against a list of PVCs, assuming all the os-images are in one namespace saves us the trouble of going over all the PVCs in the cluster or queering for individual PVC for each OS.

@glekner @rawagner - 
should we switch to fetching all PVCs in all nameapces ?
do we have a way, while in a component, to query for individual PVC per OS so we will not have to get a list of all PVCs and search for it in our code ?

Reply:
we can do a request for specific PVC with given name/namespace once user picks OS.

Reply:
>we can do a request for specific PVC with given name/namespace once user picks OS.

we do that, the problematic places are the dropdowns we show before selecting an OS in this dropdowns we show a list of all OS with a note saying if we found a matching PVC or not, to solve this we need to loop over each OS and query if it has a matching PVC, It's hard to do using hooks and/or firehose ... @rawagner can you think of a way to implement this dropdowns in a nicer way ?